### PR TITLE
#274 IE11 specific bugs

### PR DESCRIPTION
**Pull request by nickdos**
See #274 & #310

### CODE REVIEW

File: grails-app/assets/javascripts/search.js [hunk 1]
```diff
@@ -256,7 +256,7 @@ $(document).ready(function() {
         var index = i; // keep a copy
         var queryContextParam = (BC_CONF.queryContext) ? "&qc=" + BC_CONF.queryContext : "";
         var jsonUri = BC_CONF.biocacheServiceUrl + "/occurrences/search.json?q=lsid:" + lsid + "&" + BC_CONF.facetQueries +
-            "&facets=raw_taxon_name&pageSize=0&flimit=" + maxFacets + queryContextParam + "&callback=?";
+            "&facets=raw_taxon_name&pageSize=0&flimit=" + maxFacets + queryContextParam;  // + "&callback=?";
```
Comment (ansell) [inline grails-app/assets/javascripts/search.js]: Is the lack of a 'callback' parameter automatically detected to switch from JSONP to JSON in this case?

File: grails-app/views/layouts/generic.gsp [hunk 2]
```diff
@@ -22,9 +22,9 @@
         // initialise plugins
         jQuery(function(){
             // autocomplete on navbar search input
-            jQuery("form#search-form-2011 input#search-2011, form#search-inpage input#search, input#search-2013").autocomplete('http://bie.ala.org.au/search/auto.jsonp', {
+            jQuery("form#search-form-2011 input#search-2011, form#search-inpage input#search, input#search-2013").autocomplete('http://bie.ala.org.au/search/auto.json', {
```
Comment (ansell) [inline grails-app/views/layouts/generic.gsp]: Parameterising the bie URL here would be useful so we can switch to the test system easily (This is a broad issue for this URL across ALA websites but we can still fix it one instance at a time)

**Comment by nickdos** [comment]
> There are still `callback` parameters in `grails-app/assets/javascript/search.js` Is the plan to gradually remove JSONP or attempt to eliminate it on this run through?

Thanks @ansell I missed those - will fix and resubmit PR.

File: grails-app/assets/javascripts/search.js (hunk 1 above)
Comment (nickdos) [reply grails-app/assets/javascripts/search.js]: In this case, kind of... the problem wasn't JSONP but the service was still appending the callback to the request, regardless of whether JSONP was being requested. This resulted in invalid JSON. Removing the callback params fixed the invalid JSON.

File: grails-app/views/layouts/generic.gsp (hunk 2 above)
Comment (nickdos) [reply grails-app/views/layouts/generic.gsp]: Yes, should be - will fix

Comment (nickdos) [reply grails-app/views/layouts/generic.gsp]: Although this view is not used anymore (even by LA hubs) - maybe it should be deleted?

Comment (ansell) [reply grails-app/views/layouts/generic.gsp]: Deleting is always useful when things drop out of use. When I was working on the hubs it was confusing having multiple views and only one of them was able to be used and if you got it wrong it was a bit confusing but it didn't necessarily break everything.

Comment (nickdos) [reply grails-app/views/layouts/generic.gsp]: deleted

**Pull request merged**